themes/ddbasic/sass/components/node/event.scss

```diff
@@ -305,6 +305,32 @@
         }
       }
     }
+    .field-name-field-place2book-tickets {
+      position: relative;
+      width: 100%;
+      float: left;
+      .place2book-label {
+        @include font('base-bold');
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100px;
+
+        // Tablet
+        @include media($tablet) {
+          @include span-columns(2 of 5);
+        }
+      }
+      .ticket-info {
```

Comment:
In the spirit of improved CSS then how about naming `.place2book-label` and `.ticket-info` such that is is clear they are part of the same component and we can reduce nesting e.g. `place2book-tickets-label` and `place2book-tickets-info`?